internal/server/v2/facet/series.go

```diff
@@ -64,25 +64,55 @@ func SeriesFacet(
 		if err != nil {
 			return nil, err
 		}
+		// map of variable to entity to facetId to facet info from all the bt tables
+		varEntityFacets := map[string]map[string]map[string]*pb.PlaceVariableFacet{}
 		for _, btData := range btDataList {
 			for _, row := range btData {
 				entity, variable := row.Parts[0], row.Parts[1]
+				if _, ok := varEntityFacets[variable]; !ok {
+					varEntityFacets[variable] = map[string]map[string]*pb.PlaceVariableFacet{}
+				}
+				if _, ok := varEntityFacets[variable][entity]; !ok {
+					varEntityFacets[variable][entity] = map[string]*pb.PlaceVariableFacet{}
+				}
+				facetList := row.Data.(*pb.PlaceVariableFacets).GetPlaceVariableFacets()
+				for _, placeVarFacet := range facetList {
+					facetID := util.GetFacetID(placeVarFacet.Facet)
+					seenPlaceVarFacet, ok := varEntityFacets[variable][entity][facetID]
+					// If we've seen this facet already and it has the EarliestDate field,
+					// don't override the mapped facet info.
+					if ok && seenPlaceVarFacet.EarliestDate != "" {
```

Comment:
why only checking EarliestDate?

Reply:
there were cases where multiple BT versions will return the same facet, but only one of the versions has the earliest date field populated, so we want to keep the one with the earliest date field

Reply:
That seems to be a bug in Flume..